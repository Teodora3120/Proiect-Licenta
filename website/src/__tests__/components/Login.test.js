import React from 'react'
import { shallow } from 'enzyme'
import Login from '../../views/examples/Login'
it('should test Login component', () => {
  const wrapper = shallow(<Login />)
  expect(wrapper).toMatchSnapshot()
})
