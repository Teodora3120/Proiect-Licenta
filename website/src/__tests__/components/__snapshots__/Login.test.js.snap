// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test Login component 1`] = `
<Fragment>
  <Col
    lg="5"
    md="7"
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
  >
    <Card
      className="bg-secondary shadow border-0"
      tag="div"
    >
      <CardBody
        className="px-lg-5 py-lg-5"
        tag="div"
      >
        <div
          className="text-center text-muted mb-4"
        >
          <h3>
            Sign in with credentials
          </h3>
        </div>
        <Form
          role="form"
          tag="form"
        >
          <FormGroup
            className="mb-3"
            tag="div"
          >
            <InputGroup
              className="input-group-alternative"
              tag="div"
            >
              <InputGroupAddon
                addonType="prepend"
                tag="div"
              >
                <InputGroupText
                  tag="span"
                >
                  <i
                    className="ni ni-email-83"
                  />
                </InputGroupText>
              </InputGroupAddon>
              <Input
                autoComplete="new-email"
                onChange={[Function]}
                placeholder="Email"
                type="email"
              />
            </InputGroup>
          </FormGroup>
          <FormGroup
            tag="div"
          >
            <InputGroup
              className="input-group-alternative"
              tag="div"
            >
              <InputGroupAddon
                addonType="prepend"
                tag="div"
              >
                <InputGroupText
                  tag="span"
                >
                  <i
                    className="fas fa-eye c-pointer"
                    onClick={[Function]}
                  />
                </InputGroupText>
              </InputGroupAddon>
              <Input
                autoComplete="new-password"
                onChange={[Function]}
                placeholder="Password"
                type="password"
              />
            </InputGroup>
          </FormGroup>
          <div
            className="custom-control custom-control-alternative custom-checkbox"
          >
            <input
              className="custom-control-input"
              id=" customCheckLogin"
              type="checkbox"
            />
            <label
              className="custom-control-label"
              htmlFor=" customCheckLogin"
            >
              <span
                className="text-muted"
              >
                Remember me
              </span>
            </label>
          </div>
          <div
            className="text-center"
          >
            <Button
              className="my-4"
              color="primary"
              onClick={[Function]}
              tag="button"
              type="submit"
            >
              Sign in
            </Button>
          </div>
        </Form>
      </CardBody>
    </Card>
  </Col>
</Fragment>
`;
